android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -1,5 +1,4 @@
-// Copyright 2022 Google LLC
-// SPDX-License-Identifier: BSD-2-Clause
+// Copyright 2022 Google LLC SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Is this reformatting change intentional?

Reply:
Sorry no, reverted.